src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -422,9 +423,11 @@ export const Checkout = ({
         return
       }
     } catch (error) {
-      if (
-        !isManualReviewRequired((error.graphQLErrors || []) as GraphQLError[])
```

Comment:
@mehdielket I think this little `!` is the thing that made the error modal pop up when we didn't expect it to? But maybe you already figured that out 😊